app/commands/v2/unpublish.rb

```diff
@@ -132,7 +132,9 @@ def find_unpublishable_content_item(content_id)
         end
 
         filter = ContentItemFilter.new(scope: ContentItem.where(content_id: content_id))
-        filter.filter(locale: locale, state: allowed_states).first
```

Comment:
That would also make sense. I suppose we'd have to supersede on an "allow_draft" unpublishing too, in that case. Should I close this in favour of that ticket?

This is a blocker for the SP team currently, so we might need to chat about prioritising it.